examples/evaluation/huggingface-evaluation.ipynb

```diff
@@ -0,0 +1,1340 @@
+      "Downloading artifacts: 100%|██████████| 13/13 [00:07<00:00,  1.68it/s] \n",
+      "2023/12/01 15:23:36 WARNING mlflow.transformers: Could not specify device parameter for this pipeline type\n",
+      "2023/12/01 15:23:37 INFO mlflow.models.evaluation.base: Evaluating the model with the default evaluator.\n",
+      "2023/12/01 15:23:37 INFO mlflow.models.evaluation.default_evaluator: Computing model predictions.\n",
+      "Token indices sequence length is longer than the specified maximum sequence length for this model (789 > 512). Running this sequence through the model will result in indexing errors\n",
+      "Your max_length is set to 200, but your input_length is only 161. Since this is a summarization task, where outputs shorter than the input are typically wanted, you might consider decreasing max_length manually, e.g. summarizer('...', max_length=80)\n",
+      "2023/12/01 15:23:57 INFO mlflow.models.evaluation.default_evaluator: Testing metrics on first row...\n",
+      "100%|██████████| 1/1 [00:10<00:00, 10.44s/it]\n",
+      "100%|██████████| 1/1 [00:06<00:00,  6.77s/it]\n",
+      "2023/12/01 15:24:14 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: token_count\n",
+      "2023/12/01 15:24:14 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: toxicity\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: flesch_kincaid_grade_level\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: ari_grade_level\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: rouge1\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: rouge2\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: rougeL\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating builtin metrics: rougeLsum\n",
+      "2023/12/01 15:24:15 INFO mlflow.models.evaluation.default_evaluator: Evaluating metrics: answer_correctness\n",
+      "100%|██████████| 10/10 [00:10<00:00,  1.05s/it]\n",
+      "2023/12/01 15:24:26 INFO mlflow.models.evaluation.default_evaluator: Evaluating metrics: answer_quality\n",
+      "100%|██████████| 10/10 [00:10<00:00,  1.07s/it]\n"
+     ]
+    }
+   ],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "with mlflow.start_run():\n",
+    "    results = mlflow.evaluate(\n",
+    "        model_info.model_uri,\n",
+    "        eval_df.head(10),\n",
+    "        model_type=\"text-summarization\",\n",
+    "        targets=\"highlights\",\n",
+    "        extra_metrics=[answer_correctness_metric, answer_quality_metric],\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## View results"
```

Comment:
Can an explanation of what these metrics are be given (or very short explanations with relevant links to sources that will fully explain them)?